pkg/provisioner/ironic/ironic.go

```diff
@@ -376,6 +376,7 @@ func (p *ironicProvisioner) ValidateManagementAccess(data provisioner.Management
 				Name:                ironicNodeName(p.objectMeta),
 				DriverInfo:          driverInfo,
 				DeployInterface:     p.deployInterface(data),
+				FirmwareInterface:   bmcAccess.FirmwareInterface(),
```

Comment:
This cannot be done unconditionally: you need to check if the support is there.

Reply:
done